### PR TITLE
feat: Add the feature of Teacher Detail

### LINKED ISSUE #8: Admin/Teacher can see Teacher detail
- [x] CREATE TeacherMore Model
  - [x] CHANGE TeacherRegisterationView accordingly
- [x] FIX old Tests for TeacherRegisterationView
- [x] WRITE Tests for ListOneTeacherView
- [x] #23

### PR DESCRIPTION
**Pull request by ahzamm**

### CODE REVIEW

**Comment by ahzamm** [comment]
Add Tests for ListOneTeacher

File: online_school/accounts/views/teacher_views.py [hunk 1]
```diff
@@ -113,3 +116,23 @@ def post(self, request):
             {"msg": PASSWORD_CHANGE_SUCCESS_MESSAGE},
             status=PASSWORD_CHANGE_SUCCESS_STATUS,
         )
+
+
+# @swagger_auto_schema(responses=List_one_student_response)
```
Comment (ahzamm) [inline online_school/accounts/views/teacher_views.py:115]: ADD Docs for this view

File: online_school/accounts/models/teacher_models.py [hunk 2]
```diff
@@ -30,6 +31,27 @@ def create_user(self, email=None, name=None, password=None, **kwargs):
...
 
+class TeacherMore(models.Model):
+
+    user = models.OneToOneField(
+        "Teacher",
+        related_name="teacher_more_info",
+        on_delete=models.CASCADE,
+    )
+    tea_id = models.CharField(max_length=20, unique=True)
+    salary = models.IntegerField()
```
Comment (ahzamm) [inline online_school/accounts/models/teacher_models.py]: add null=True to these fields

File: online_school/accounts/models/teacher_models.py [hunk 3]
```diff
@@ -30,6 +31,27 @@ def create_user(self, email=None, name=None, password=None, **kwargs):
...
+    contact_number = models.IntegerField()
```
Comment (ahzamm) [inline online_school/accounts/models/teacher_models.py]: add null=True

File: online_school/accounts/models/teacher_models.py [hunk 4]
```diff
@@ -30,6 +31,27 @@ def create_user(self, email=None, name=None, password=None, **kwargs):
...
+    degree = models.CharField(max_length=20)
```
Comment (ahzamm) [inline online_school/accounts/models/teacher_models.py]: null=True

**Comment by AhzamAhmed6** [comment]
- [ ] Change TeacherRegisterationView accordingly
- [ ] Fix Tests for TeacherRegisterationView
- [ ] Write Tests for ListOneTeacherView

**Pull request merged**